README.md

```diff
@@ -7,11 +7,26 @@
 
 > Self identifying base encodings
 
-Multibase is a protocol for distinguishing base encodings and other simple string encodings, and for ensuring full compatibility with program interfaces. It answers the question:
+Multibase is a protocol for distinguishing base encodings of bytes in text.
```

Comment:
Wouldn't be "base encodings of string" clearer than "base encodings of bytes in text"?

Reply:
Well, the "string" wasn't encoded in a base. That is, we've encoded some *byte* string into a textual representation using a base encoding.

Reply:
But yeah, this could be improved (I just want it to be unambiguous).

Reply:
I've rewritten that sentence again. Is it clear or still confusing (I'm having trouble coming up with a wording that's both technically accurate and understandable).